packages/portal-proto/src/features/facets/NumericRangeFacet/FromTo.tsx

```diff
@@ -0,0 +1,309 @@
+import { useState } from "react";
+import { DAYS_IN_YEAR } from "@gff/core";
+import { NumberInput, SegmentedControl } from "@mantine/core";
+import {
+  ClearFacetHook,
+  FromToRange,
+  RangeFromOp,
+  RangeToOp,
+  UpdateFacetFilterHook,
+} from "../types";
+import {
+  adjustDaysToYearsIfUnitsAreYears,
+  adjustYearsToDaysIfUnitsAreYears,
+  buildRangeOperator,
+  getLowerAgeYears,
+} from "../utils";
+import { RiErrorWarningFill as ErrorWarningIcon } from "react-icons/ri";
+import { MdWarning as WarningIcon } from "react-icons/md";
+import { useDeepCompareEffect } from "use-deep-compare";
+import { useForm } from "@mantine/form";
+
+interface FromToProps {
+  readonly minimum: number;
+  readonly maximum: number;
+  readonly units?: string;
+  readonly field: string;
+  readonly useClearFilter: ClearFacetHook;
+  readonly useUpdateFacetFilters: UpdateFacetFilterHook;
+  readonly values?: FromToRange<number>;
+  readonly changedCallback?: () => void;
+  readonly clearValues?: boolean;
+}
+
+const WARNING_DAYS = Math.floor(90 * DAYS_IN_YEAR);
+
+const applyButtonClasses = `
+  flex
+  flex-row
+  subpixel-antialiased
+  rounded-md
+  text-base-max
+  font-montserrat
+  font-medium
+  text-sm
+  transition
+  w-full
+  justify-center
+  align-center
+  py-1
+  bg-primary-dark
+  hover:bg-primary-darkest
+  hover:shadow-[0_4px_5px_0px_rgba(0,0,0,0.35)]
+`;
+
+interface WarningOrErrorProps {
+  hasErrors: boolean;
+  isWarning: boolean;
+  lowerUnitRange: number;
+  upperUnitRange: number;
+}
+
+const WarningOrError: React.FC<WarningOrErrorProps> = ({
+  hasErrors,
+  isWarning,
+  lowerUnitRange,
+  upperUnitRange,
+}: WarningOrErrorProps) => (
+  <div className="flex flex-col gap-1 my-1">
+    {hasErrors ? (
+      <span className="text-[#AD2B4A]">
```

Comment:
We define the tailwind color "utility-error" which would be better to use.

Reply:
I updated this.